helpers/test/prism.vdiff.js

```diff
@@ -0,0 +1,102 @@
+import '../../components/typography/typography.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+import { classMap } from 'lit/directives/class-map.js';
+import { formatCodeElement } from '../prism.js';
+
+const jsFunction =
+`function helloGrumpy(name) { /* ... */ }
+helloGrumpy('Wizard');`;
+const jsFullFunction =
+`function helloGrumpy(name) {
+	console.log(\`Hi there \${name}.\`);
+}
+helloGrumpy('Wizard');`;
+
+function createCodeWrapper({ content, dark, lineNumbers, language, inline }) {
+	const classes = {
+		'd2l-code': true,
+		'd2l-code-dark': dark,
+		'line-numbers': lineNumbers
+	};
+	if (inline) classes[`language-${language}`] = true;
+	/*return !inline ? html`
+		<div class="d2l-typography" style="max-width: 500px;">
+			<pre class="${classMap(classes)}"><code class="language-${language}">${content}</code></pre>
+		</div>
+	` : html`
+		<div class="d2l-typography" style="width: 400px;">
+			<p>Some inline code... <code class="${classMap(classes)}">${content}</code> Mmmm, yummy code.</p>.
+		</div>
+	`;*/
+	return !inline ? html`
+		<div style="max-width: 500px; letter-spacing: normal;">
+			<pre class="${classMap(classes)}"><code class="language-${language}">${content}</code></pre>
+		</div>
+	` : html`
+		<div style="width: 400px; color: black; font-family: auto; font-size: 20px; letter-spacing: normal;">
+			<p>Some inline code... <code class="${classMap(classes)}">${content}</code> Mmmm, yummy code.</p>.
+		</div>
+	`;
+}
+
+describe('prism-helper', () => {
+
+	describe('general', () => {
+		[
+			{ name: 'block', content: jsFullFunction, language: 'javascript' },
+			{ name: 'block-line-numbers', content: jsFullFunction, language: 'javascript', lineNumbers: true },
+			{ name: 'inline', content: 'const jelly = \'donuts\';', language: 'javascript', inline: true }
+		].forEach(({ name, content, language, lineNumbers, inline }) => {
+			['light', 'dark'].forEach(color => {
+				it(`${name}-${color}`, async() => {
+					const elem = await fixture(createCodeWrapper({ content, dark: color === 'dark', language, lineNumbers, inline }));
+					window.codeFormatted = await formatCodeElement(elem.querySelector('.d2l-code'));
+					await expect(elem.querySelector('pre, p')).to.be.golden();
+				});
+			});
+		});
+	});
+
+	describe('tokens', () => {
+		[
```

Comment:
I left this in the original order to make it easier to compare to the old test, but I could also rearrange so languages are together.